requirements.txt

```diff
@@ -3,12 +3,11 @@ gevent==1.4.0
 gevent-websocket==0.10.1
 h5py==2.6.0
 python-socketio==4.4.0
-numpy==1.13.1
-Pillow==2.2.1
+numpy==1.16.6
+Pillow==6.2.2
 scipy==0.19.1
 Markdown==3.1.1
-setuptools==36.7.2
-keras==2.0.8
-tensorflow==1.3.0
+setuptools==41.0.0
+tensorflow==1.15
```

Comment:
One of the [project instruction](https://classroom.udacity.com/nanodegrees/nd013/parts/01a340a5-39b5-4202-9f89-d96de8cf17be/modules/702b3c5a-b896-4cca-8a64-dfe0daf09449/lessons/e43b2e6d-6def-4d3a-b332-7a58b847bfa4/concepts/3251f513-2f82-4d5d-88b6-9d646bbd9101) states that:
_Please note that we will not be able to accomodate special launch instructions or run additional scripts from your submission to download files - this includes any changes to the requirements.txt file._

Does that mean reviewer/Carla will not be able to run our model if it is trained under a different version?

Reply:
@CaoFM Oh... good point.

Looks like this will be a solution: https://stackoverflow.com/questions/53927700/how-to-use-object-detection-api-with-an-old-version-of-tensorflow-v1-3-0